gui/wxpython/datacatalog/tree.py

```diff
@@ -693,11 +693,10 @@ def OnCreateLocation(self, event):
                                           self.selected_grassdb[0].data['name'])
         )
         if location is not None:
+            if user_mapset != "PERNAMENT":
+                self.InsertMapset(name=user_mapset, location_node=location)
```

Comment:
Why to InsertMapset here and why only when not PERMANENT (note the spelling, BTW)? ReloadTreeItems is called afterwards. The bigger problem is the database.

Reply:
User mapset was solved within create_location_interactively.

Reply:
Spelling corrected.